---
title: "Eco-Evo Genomics of aging rate in wild populations"
summary: " "
abstract: ""
date: 2021-09-01
image: 
  caption: ''
  focal_point: Smart
---

It is challenging to identify and track the genetic architecture of a complex trait in a wild outbreeding population. However, it is possible to take advantage of -omic advances, traditional genetic methods, and computational tools to push this interesting frontier. In this project, we focus on the same pools sampled for environmental conditions. Thus, for each species we compare arid vs humid fish in their rate of genomic and functional changes over three years. We collect fish at three time points along the desiccation timeline to provide multiple test tissues. Using differential gene expression for arid-wet conditions, whole-genome sequencing, genome-wide association mapping (GWAS), along with functional trait measurements (changes in tissue microbiomes across season, accumulation of tissue damage indicators, and swimming performance such as fatigue and swim distance), the goal is to understand the profile of network adaptation and allele frequency variation. These data will serve as the basis for comparison with similar data sets generated in in controlled conditions. Apart from gaining insight into the ‘natural’ mechanism of differential aging in the wild, findings are expected to illuminate the problem of low corroboration of lab findings in natural populations.

---
